### ISSUE TITLE
Enhancement: Add context to cart quantity

### ISSUE THREAD

**Issue by mikeyarce** [woocommerce/woocommerce#16966]
When using a screen reader, it's important to add context to form fields, especially in a table.
In the cart, our quantity row reads "1", but it doesn't have context as to what that 1 is about.

In this example:
https://webaim.org/techniques/forms/advanced#multiple

You can see that adding an `aria-labelledby="product-name quantity"` would be the best way to make our cart quantity field accessible.  That would help people know that this number is for a specific product, and for the quantity they want of that product.

**Comment by nishitlangaliya** [comment]
As per suggestions I have added aria-labelledby quantity field for cart page.

Thanks.

**Comment by mikeyarce** [comment]
@nishitlangaliya sorry - that was just an example, it was not meant to be actually `product-name` and `quantity` but contextual words to describe what is going on.

For example, if I have 2 Backpacks in my cart, the label should read `aria-labelledby="backpack quantity-2"`.

**Comment by nishitlangaliya** [comment]
As per suggestion aria-labelledby changes done. Please check for the same and let me know if required any changes .

Thanks.

**Comment by nishitlangaliya** [comment]
As per suggestions New PR has done https://github.com/woocommerce/woocommerce/pull/17006
